web/Views/Shared/_Layout.cshtml

```diff
@@ -1,102 +1,104 @@
-                    @if (User.IsInRole("Admin"))
-                    {
-                        <li class="nav-item dropdown">
-                            <a class="nav-link active dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-bs-toggle="dropdown">
-                                Admins
-                            </a>
-                            <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
-                                <a class="dropdown-item text-dark" asp-area="" asp-controller="Admin" asp-action="GetAdminList">Admin List</a>
-                                <a class="dropdown-item text-dark" asp-area="" asp-controller="Admin" asp-action="LandlordList">Landlord List</a>
-                            </div>
-                        </li>
-                    }
-                    <li class="nav-item">
-                        <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="ContactUs">Contact Us</a>
-                    </li>
-                    @if (User.IsInRole("Landlord"))
-                    {
-                        <li class="nav-item">
-                            <a class="nav-link text-dark" asp-area="" asp-controller="Landlord" asp-action="MyProfile">Profile</a>
-                        </li>
-                        <li class="nav-item">
-                            <a class="nav-link text-dark" asp-area="" asp-controller="Landlord" asp-action="ViewProperties">View Properties</a>
-                        </li>
-                    }
-                </ul>
 
-                <ul class="navbar-nav ms-auto">
+            <div class="collapse navbar-collapse">
+                <div class="navbar-nav me-auto">
+                    @* Change this link to landlord index when created *@
+                    <a class="nav-link text-dark mx-3" asp-controller="Landlord" asp-action="RegisterGet">List your properties</a>
+                    <a class="nav-link text-dark mx-3" asp-controller="Admin" asp-action="Index">Facilitate rentals</a>
+                </div>
+
+                <div class="navbar-nav">
                     @if (User.Identity?.IsAuthenticated == true)
                     {
-                        <li class="nav-item">
-                            <a class="nav-link">@(((BricksAndHeartsUser)User.Identity).GoogleEmail)</a>
-                        </li>
-                        <li class="nav-item">
-                            @using (Html.BeginForm("Logout", "Login", FormMethod.Post))
-                            {
-                                <button type="submit" class="btn btn-link nav-link border-0">Log out</button>
-                            }
-                        </li>
+                        @if (User.IsInRole("Landlord"))

```

Comment:
Am I being silly or does this mean admins won't have a profile picture?

Reply:
That's a good point. I think I left it off at the minute because there's no admin profile page, and because this icon is very temporary anyway. I can add one in for admins if you think it's useful?

Reply:
I think its fine for the moment, can always make another ticket if it really feels necessary